docs/current_docs/partials/_install-cli.mdx

```diff
@@ -146,3 +148,17 @@ where.exe dagger
 
 </TabItem>
 </Tabs>
+
+### Development version
+
+:::warning
+Development versions should be considered unstable and may produce unexpected results.
+:::
+
+To install the latest development version, use the following command:
+
+```shell
```

Comment:
@gerhard should we have tabs for each language? If yes, what would be the Windows equivalent for this command?

Reply:
`.\install.ps1 -DaggerCommit head`